src/main/java/domain/LottoAutoTicket.java

```diff
@@ -0,0 +1,25 @@
+package domain;
+
+public class LottoAutoTicket {
+
+    private static final int LOTTO_PRICE = 1000;
+    private int price;
+    private final int passiveLottoTicket;
```

Comment:
```suggestion
    private static final int LOTTO_PRICE = 1000;
    
    private final int passiveLottoTicket;
    
    private int price;
```

price를 필드로 가지고 있을 필요가 있을까요? 
해당 객체 입장에서는 price는 일회용인 것 같은데 매개변수로만 받아서 로또 혹은 개수만 생성해줘도 될 것 같아요~